Automattic-Tracks-iOS/Event Logging/EventLoggingUploadManager.swift

```diff
@@ -34,17 +29,18 @@ class EventLoggingUploadManager {
             return
         }
 
-        guard let fileContents = fileManager.contents(atUrl: log.url) else {
+        guard fileManager.fileExistsAtURL(log.url) else {
```

Comment:
We don't need to read the file here – the API actually handles this, so it was a waste of resources – just validating that it exist allows the delegate callbacks to handle errors in the same way.